apps/web-portal/lib/WebPortalExpressApp.mjs

```diff
@@ -1,29 +1,38 @@
 import assert from 'assert'
 import express from 'express'
-import * as RestApiV1 from './rest-api-v1'
 import {makeRunner} from '@zorko-io/util-use-case'
+import {MockLogger} from '@zorko-io/util-logger'
+import * as RestApiV1 from './rest-api-v1'
 
-// TODO: Handle process start/stop
-// - add logging
-// - replace console.log
-// label: enhancement
 export class WebPortalExpressApp {
   /**
    * @type {Express}
    */
 
+  #app = null
+
   #http = null
+
   #config = null
 
+  #process = null
+
+  #logger = null
+
   /**
    * @param {Object} context
    * @param {Object} context.config - app config
+   * @param {Object} context.process - process
```

Comment:
Just a comment
FYI: it's better to provide types (if they exist) it will bring more intelligence for IDE  code highlight 
```
   * @param {NodeJS.EventEmitter} process - blblblb
   * @param {CoreLogger} logger - blblblbl
```